src/libraries/System.IO/tests/Stream/Stream.ReadExactly.cs

```diff
@@ -0,0 +1,300 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
```

Comment:
Unintended using?

Reply:
Yes, and I can't for the life of me figure out how it got in here. It must have been an automatic "add using" from VS as I was searching for a Type while writing code.

My next question was: "why are the tests compiling on my machine?". And discovered that the tests do reference NuGet.Frameworks locally through a `<PackageReference Include="Microsoft.NET.Test.Sdk"`. However, we turn that PackageReference off in CI, which is causing it to work locally, but fail in CI:

https://github.com/dotnet/runtime/blob/951df04d97405ff6dd13e708873549fb9acf9a6c/eng/testing/xunit/xunit.props#L17-L19